code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -66,10 +66,26 @@
 	PH?.Trigger()
 
 /datum/species/psyphoza/get_species_description()
-	return "..."
+	return "Psyphoza are a species of extra-sensory lesser-sensory fungal, \
+	based humanoids, infamous for their invulnerability to occlusion based \
+	magic tricks."
 
 /datum/species/psyphoza/get_species_lore()
-	return list("...")
+	return list(
+		"A standing testament to the humor of mother nature, Psyphoza have evolved powerful and mystical \
+			psychic abilities, which are almost completely mitigated by the fact they are absolutely \
+			blind, and depend entirely on their psychic abilities to navigate their surroundings.",
+
+		"Psyphoza culture is deeply rooted in superstition, mysticism, and the occult. It is their belief \
```

Comment:
Colour is not really something the Psyphoza can observe themselves, or likely even comprehend due to blindness. 

Or are we actually going with the "legally blind" thing?